packages/flame_tiled/lib/src/renderable_layers/tile_layers/isometric_tile_layer.dart

```diff
@@ -60,7 +60,7 @@ class IsometricTileLayer extends FlameTileLayer {
               .translate(spriteOffset.dx, spriteOffset.dy),
         );
         final flips = SimpleFlips.fromFlips(tileGid.flips);
-        final scale = size.x / src.width;
+        final scale = size.x / map.tileWidth;
```

Comment:
Can you update the tests to include a larger tile? Does this still pack well for faster rendering?

Reply:
I'll be honest, I didn't dig into the inner working of flame_tiled much beyond this little bit of the code to figure out why my sample tiles weren't rendering as expected. I threw up a PR with the fix since I'd found it, but if there's other things this might impact I don't know enough to comment other than to say that this should be no different for cases where the native image size matches the map tile size (which I assume is all current uses).

I took another quick look just now and realized that all the other types of tile_layers seems to actually have this same bug, so maybe those should be updated as well? I don't have test cases for them, though, and while I took a quick look at the existing tests I couldn't figure out enough to know where/how was the right way to add a test for this.

Reply:
Here's my take: if none of the other tests exploded, you're good!  I'm just asking for a simple test that shows your expected results to prevent regressions. That could just be using a small map with your one oversized tile and making a golden image.

Reply:
Tests added and included fixes for the other layer types as well. Hopefully I did that right!